docs/30_getting-started-guide/index.md

```diff
@@ -28,45 +43,45 @@ In this section, we describe the base components and protocols used in the EOSIO
 
 ## Configuration Guides
 
-The configuration guides section will show you how to configure [Nodeos.](../../glossary/index#nodeos) How [Nodeos](../../glossary/index#nodeos) is configured will depend on what your desired usage is. [Nodeos](../../glossary/index#nodeos) configuration controls which [plugins](../../glossary/index#plugin) are used and how those [plugins](../../glossary/index#plugin) are configured. The [plugins](../../glossary/index#plugin) are used to specify [Nodeos](../../glossary/index#nodeos) behaviour for specific purposes. 
+The configuration guides section will show you how to configure [Nodeos.](../glossary/index/#nodeos) How `Nodeos` is configured will depend on what your desired usage is. `Nodeos` configuration controls which [plugins](../glossary/index/#plugin) are used and how those `plugins` are configured. The `plugins` are used to specify `Nodeos` behaviour for specific purposes. 
```

Comment:
Sentence is clumsy, How about ->

The configuration guides section will show you how to configure [Nodeos.](../glossary/index/#nodeos) How `Nodeos` is configured will depend on your desired usage.

Reply:
Applied suggestion (this part of the content was not changed in my PR. It may need another round of review in a fresh PR).